identity/controller.go

```diff
@@ -1,17 +1,17 @@
 package identity
 
 import (
-	"github.com/mysterium/node/service_discovery/dto"
 	"errors"
+	"github.com/mysterium/node/service_discovery/dto"
 )
 
 type identityHandler struct {
-	manager *identityManager
+	manager *IdentityManager
 	cache   *identityCache
 }
 
-func SelectIdentity(dir string, nodeKey string) (id *dto.Identity, err error) {
-	handler := NewIdentityHandler(dir)
+func SelectIdentity(keystore keystoreInterface, nodeKey string) (id *dto.Identity, err error) {
```

Comment:
passing key store to select identity looks bad, shouldn't this function be part of identitymanager? I hope it's in the middle of refactoring

Reply:
@ignasbernotas is taking care of this from here afaik.